Modules/TPC/run/tpcQCTrending_laserCalib.json

```diff
@@ -0,0 +1,118 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
```

Comment:
We used to have this url everywhere but it spams the DNS for no useful reason. Could you just use an empty url ? 
Thank you

Reply:
I changed the url to an empty string `""`. Is this right?

Reply:
yes that's good